CONTRIBUTING.md

```diff
@@ -1,41 +1,58 @@
-  * One of the main issues with the current code is that a lot of it is very intertwined (circular imports, lots of classes constructed with the entired recipe database as an argument, etc). I've tried to implement a couple of unit tests to test single features (which, as the name says, are supposed to act on "units of work") -- e.g. some website importers. That's currently not easy at all, as I need to initialize and pass either a dummy database, or replace some classes they would normally depend on with dummy classes. So we need more "atomic" classes before being able to write proper tests.
-* We could also use some more documentation, both for users and for developers (i.e. by adding comments to the source code).
-* We'd like to turn our current [web page](http://thinkle.github.com/gourmet/) into an actual web _site_. To work on the web site, just fork our repo and check out the gh-pages branch.
- * It should retain most of the current visual elements (icon, color scheme, Lobster font)
- * but use [Jekyll](https://help.github.com/articles/using-jekyll-with-pages) (or maybe the [Octopress](http://octopress.org/) framwork for Jekyll; and maybe [Disqus](http://disqus.com/) for comments) to provide a news blog into which we'll then feed our old news items back from sourceforge times, and that we're going to use for future announcements (e.g. for releases).
- * What about a download with e.g. a [Download for Ubuntu](https://wiki.ubuntu.com/SoftwareCenter/AppPromotion) button?
- * A "Star on GitHub" button (like http://docs.python-requests.org/en/latest/ has) would be nice.
- * Some way to donate would be also nice (PayPal, gittip.com, Flattr,...?)
-* Bug triaging. We have about [200 open issues](https://github.com/thinkle/gourmet/issues), and some of them are a couple of years old and have already been fixed. Some help in tagging, commenting, and ideally closing bugs is very welcome! (Note that to close a bug, you should really be sure it's okay to do so because you have tested Gourmet to work under the same conditions the bug reporter did.)
+# Getting Started
+Install Gourmet's dependencies:
+
+    pip3 install -r requirements.txt
+    pip3 install -r development.txt
+
+`requirements.txt` contains the dependencies needed for Gourmet itself,
+`development.txt` contains the packages required for testing.
+
+Before you start development, you should first build localized *.mo and
+*.gourmet-plugin files within a build/ subdirectory of the source tree by
+running
+
+    python3 setup.py build_i18n -m
```

Comment:
Why do we need `-m` in the end?

Reply:
Ah, nice catch, it's a relic from the previous implementation. I've reworked the internationalisation, but I didn't update that line of doc...